### DISCUSSION TITLE
<title>education Package

### DISCUSSION THREAD

**Discussion by znu8r** [community/community#40567] [GitHub Education]
### Select Topic Area

Question

### Body

Hello, I have a problem with my student package only My university ID card and when I send it, it is rejected

**Comment by samus-aran** [comment]
Hi @znu8r  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
